src/Microsoft.OData.Core/ODataWriter.cs

```diff
@@ -53,6 +53,47 @@ public ODataWriter Write(ODataResourceSet resourceSet, Action nestedAction)
         public abstract Task WriteStartAsync(ODataResourceSet resourceSet);
 #endif
 
+        /// <summary>Starts the writing of a delta resource set.</summary>
+        /// <param name="deltaResourceSet">The resource set or collection to write.</param>
+        public virtual void WriteStart(ODataDeltaResourceSet deltaResourceSet)
+        {
+            // todo: replace with better error message

```

Comment:
>todo [](start = 15, length = 4)

We can just use a string literal w/o localization support here.

Reply:
Yea; I actually think NotImplementedException is probably fine.  Service can catch and return 503 Not Implemented.

---
In reply to: [150167763](https://github.com/OData/odata.net/pull/989#discussion_r150167763) [](ancestors = 150167763)